src/test/cli/ceph-conf/help.t

```diff
@@ -18,7 +18,9 @@
                                     can be opened in the resulted comma
                                     delimited search list.
     -D|--dump-all                   dump all variables.
-  
+    --show-config-value <key>       Print the corresponding ceph.conf value
+                                    that matches the specified key. Also searches
+                                    global defaults.
```

Comment:
```diff
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/ceph-conf/help.t: failed
--- /home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/ceph-conf/help.t
+++ /home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/ceph-conf/help.t.err
@@ -21,6 +21,7 @@
     --show-config-value <key>       Print the corresponding ceph.conf value
                                     that matches the specified key. Also searches
                                     global defaults.
+  
   FLAGS
     --name name                     Set type.id
     [-s <section>]                  Add to list of sections to search
```

Reply:
@falcon78921  i am sorry for the bike-shedding. i think we need to apply following patch on top of your change to address this failure.
```diff
diff --git a/src/test/cli/ceph-conf/help.t b/src/test/cli/ceph-conf/help.t
index bdbde0a9ea..8fa2152d39 100644
--- a/src/test/cli/ceph-conf/help.t
+++ b/src/test/cli/ceph-conf/help.t
@@ -21,6 +21,7 @@
     --show-config-value <key>       Print the corresponding ceph.conf value
                                     that matches the specified key. Also searches
                                     global defaults.
+  
   FLAGS
     --name name                     Set type.id
     [-s <section>]                  Add to list of sections to search
```
please note the two spaces before the `FLAGS` line. that's because the default indent of [cram](https://bitheap.org/cram/) is 2.

Reply:
@tchaikov thanks for the patch :)